packages/storage-types/index.d.ts

```diff
@@ -16,10 +16,10 @@
  */
 
 import { FirebaseApp } from '@firebase/app-types';
+import { StorageError as FirebaseStorageError } from '@firebase/storage';
```

Comment:
I'm not sure if this will work, as (1) `@firebase/storage` isn't a dependency of `@firebase/storage-types` so it's not great to import from it. In practice it should always be there because users install all of Firebase at once, but it doesn't seem right and I think some of the newer package managers yell about this. (2) I think this is sort of exporting a class from a `d.ts` file which I don't think you can do, not 100% sure, maybe it just treats it as a type. I know we were importing FirebaseError before which is a class but it was only used as an `extends` for an interface, and that interface was what's exported. Hm, is there any way to make an interface from scratch here that matches the StorageError class?